sql/moz-fx-data-shared-prod/search_terms_derived/sanitation_job_metadata/metadata.yaml

```diff
@@ -0,0 +1,31 @@
+---
+friendly_name: Sanitized Firefox Search Terms
+owners:
+  - ctroy@mozilla.com
+labels:
+  incremental: true
+description: |-
+  A sanitized variant of search terms submitted to Firefox.
+
+  This table allows removal of PII and slightly longer retention than
+  the underlying ping table.
+scheduling:
+  dag_name: bqetl_search_terms_daily
+  arguments: ['--schema_update_option=ALLOW_FIELD_ADDITION']
+bigquery:
+  time_partitioning:
+    field: submission_timestamp
```

Comment:
As discussed above there's no `submission_timestamp` field for this table.

Reply:
Changed to timestamp 😅